Content.Shared/Movement/Systems/SharedMoverController.Input.cs

```diff
@@ -373,15 +373,17 @@ private void HandleRunChange(EntityUid uid, ushort subTick, bool walking)
             SetSprinting((uid, moverComp), subTick, walking);
         }
 
-        public (Vector2 Walking, Vector2 Sprinting) GetVelocityInput(InputMoverComponent mover)
+        // ES START
+        // forceWalk
+        public (Vector2 Walking, Vector2 Sprinting) GetVelocityInput(InputMoverComponent mover, bool forceWalk=false)
```

Comment:
Effectively an upstream's problem nitpick: This function probably shouldn't be in charge of whether you're walking or not, and the idea of a character having two separate velocity vectors is a bit of a code smell.